carbonmark-api/src/utils/helpers/fetchProjectPoolInfo.ts

```diff
@@ -23,72 +24,144 @@ type PoolInfoMap = {
   ubo: PoolInfo;
   nbo: PoolInfo;
 };
+/**
+ * Params for fetchProjectPoolInfo
+ */
+
+type Params = {
+  projectID: string; // Project key `"VCS-981"`
+  vintage: number; // Vintage Int 2017
+};
 
 /**
  * Stats for all project tokens across both bridges
  */
+export type BigNumberStats = {
+  bridged: ethers.BigNumber;
+  retired: ethers.BigNumber;
+  totalSupply: ethers.BigNumber;
+};
+
+const initialStats: BigNumberStats = {
+  bridged: ethers.BigNumber.from(0),
+  retired: ethers.BigNumber.from(0),
+  totalSupply: ethers.BigNumber.from(0),
+};
 export type Stats = {
   totalBridged: number;
   totalRetired: number;
   totalSupply: number;
 };
 
-type Params = {
-  key: string; // Project key `"VCS-981"`
-  vintage: string; // Vintage string `"2017"`
+/** Return Types for Digital Carbon Query */
+
+type PoolBalance = {
+  balance: string;
+  id: string;
+  deposited: string;
+  redeemed: string;
+  pool: {
+    name: string;
+    supply: string;
+    id: string;
+    decimals: number;
+  };
 };
 
-const initialStats: Stats = {
-  totalBridged: 0,
-  totalRetired: 0,
-  totalSupply: 0,
+type CarbonCredit = {
+  vintage: number;
+  currentSupply: string;
+  poolBalances: PoolBalance[];
+  id: string;
+  crossChainSupply: string;
+  bridgeProtocol: string;
+  bridged: string;
+  retired: string;
 };
 
+type CarbonCredits = CarbonCredit[];
+
 /**
  * Query the subgraph for a list of the C3Ts and TCO2s that exist for a given project-vintage combination.
```

Comment:
more terminology improvements, since we are in here
```suggestion
 * Query the subgraph for a list of the C3Ts and TCO2s that exist for a credit vintage.
```